scheme-lib/rnrs.ss

```diff
@@ -115,6 +115,11 @@
   (define (cadadr p) (car (cdadr p)))
   (define (cadddr p) (car (cdddr p)))
   (define (cdaddr p) (cdr (caddr p)))
+  (define (memq x ls)
```

Comment:
It'd be good to go ahead and remove this from `compiler.ss`, but that'll need a new snapshot first. Probably good for a follow up PR.